README.md

```diff
@@ -1,34 +1,57 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
+## General overview of what this repo contains
+
+This repo contains a frontend interface built with nextJS, the UI connects to:
+- [GemFarm](https://github.com/gemworks/gem-farm) for Staking as a Service through a service layer API built by Honey Finance - WIP
+- [Honey-SDK](https://github.com/honey-labs/honey-sdk) for permissionless lending pools that use NFTs as collateral - WIP
+
 ## Getting Started
 
-First, run the development server:
+First things first,
+
+Make sure to create a `.env` file based on the `.env.example` file. The RPC_URL is the standard mainnet RPC.
+You should replace this with a custom one if you are in need of better performance.
```

Comment:
add "you might not need it if you are development on devnet but currently we do not have farm available on devnet, However we will add instructions to add farm to devnet in the future"

Reply:
good addition! will add it tomorrow.